Xwt/Xwt.Drawing/Font.cs

```diff
@@ -184,7 +182,7 @@ internal static Font FromName (string name, Toolkit toolkit)
 			if (fb != null)
 				return new Font (fb, toolkit);
 			else
-				return Font.SystemFont;
+				return null;
```

Comment:
Could this break other users of this API? I wonder if we can a get away with using no XWT in VSMac's FontService...

Reply:
you are right, maybe I could copy all of this over a standalone method instead this change which could be break other scenarios.